pennylane/pulse/parametrized_evolution.py

```diff
@@ -75,86 +72,182 @@ class ParametrizedEvolution(Operation):
-        gates are executed simultaneously, but rather both gates evaluate their respective
-        scalar-valued functions using the same time values.
+    To create a :class:`~.ParametrizedEvolution`, we first define a :class:`.~ParametrizedHamiltonian`
+    describing the system, and then pass it to :func:`~pennylane.evolve`:
 
-    .. note::
+    .. code-block:: python
 
-        To execute two :class:`ParametrizedHamiltonian` instances simultaneously one must wrap them
-        with the same ``ParametrizedEvolution`` gate.
+        from jax import numpy as jnp
 
-    **Example**
+        f1 = lambda p, t: jnp.sin(p * t)
+        H = f1 * qml.PauliY(0)
 
-    Let's first build two :class:`ParametrizedHamiltonian` objects:
-
-    >>> ops = [qml.PauliX(0), qml.PauliY(1), qml.PauliZ(2)]
-    >>> coeffs = [lambda p, t: p for _ in range(3)]
-    >>> H1 = qml.dot(coeffs, ops)  # time-independent parametrized Hamiltonian
-    >>> ops = [qml.PauliZ(0), qml.PauliY(1), qml.PauliX(2)]
-    >>> coeffs = [lambda p, t: p * jnp.sin(t) for _ in range(3)]
-    >>> H2 = qml.dot(coeffs, ops) # time-dependent parametrized Hamiltonian
-    >>> H1, H2
-    (ParametrizedHamiltonian: terms=3, ParametrizedHamiltonian: terms=3)
-
-    Now we can execute the evolution of these parametrized hamiltonians:
-
-    >>> dev = qml.device("default.qubit", wires=3)
-    >>> @qml.qnode(dev, interface="jax")
-    ... def circuit(params):
-    ...     qml.evolve(H1)(params, t=[0, 10])
-    ...     qml.evolve(H2)(params, t=[0, 10])
-    ...     return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1) @ qml.PauliZ(2))
-    >>> params = jnp.array([1., 2., 3.])
-    >>> circuit(params)
-    Array(-0.01543971, dtype=float32)
+        ev = qml.evolve(H)
 
-    We can also compute the gradient of this evolution with respect to the input parameters!
+    The initial :class:`~.ParametrizedEvolution` does not have set parameters, and so will not
+    have a matrix defined. To obtain an Operator with a matrix, it must be passed parameters and
+    a time interval:
 
-    >>> jax.grad(circuit)(params)
-    Array([ 0.6908507,  0.0865578, -1.4594607], dtype=float32)
+    >>>  qml.matrix(ev([1.2], t=[0, 4]))
+    Array([[ 0.72454906+0.j, -0.6892243 +0.j],
+       [ 0.6892243 +0.j,  0.72454906+0.j]], dtype=complex64)
 
-    As mentioned in the warning above, ``circuit`` is not executing the evolution of ``H1`` and ``H2``
-    simultaneously, but rather executing ``H1`` in the ``[0, 10]`` time window and then executing
-    ``H2`` with the same time window.
+    The parameters can be updated by calling the :class:`~.ParametrizedEvolution` again with different inputs.
 
-    If we want to execute both hamiltonians simultaneously, we need to wrap them with the same
-    ``ParametrizedEvolution`` operator:
+    Additional options regarding how the matrix is calculated can be passed to the :class:`.ParametrizedEvolution`
+    along with the parameters, as keyword arguments.
```

Comment:
Not sure what this sentence is trying to say

Reply:
Changed it to "When calling the :class:`~.ParametrizedEvolution`, keyword arguments can be passed to specify behaviour of the ODE solver."
    
Does that seem clearer, or should I just not mention that here? The `kwargs` are listed and explained at the beginning of the docstring, so maybe its not helpful here.